### PR TITLE
Add module l10n_fr_department_delivery

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: l10n_fr_department_delivery/model/delivery.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = 'delivery.carrier'
+
+    @api.multi
+    def grid_get(self, contact_id):
+        '''This is almost a copy-paste of the native grid_get() method
+        from odoo/addons/delivery/delivery.py
+        I didn't find a way to inherit this method via super() without too
```
Comment (clonedagain) [inline l10n_fr_department_delivery/model/delivery.py:40]: At least can you please mark the beginning/end of the part you changed,  so we can upgrade the copy once in a while?

File: l10n_fr_department_delivery/README.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+French Departments - Delivery
+=============================
+
+This module adds a field *French Departments* on delivery grids.
+
+Configuration
+=============
+
+To configure this module, go to *Warehouse > Configuration > Delivery > Delivery Pricelist* and, in the *Destination* tab, you will find the field *French Departments*.
+
```
Comment (clonedagain) [inline l10n_fr_department_delivery/README.rst:10]: Please indicate that this moduke is incompatible with other modules whuch extend the selection of carriers, in a "known problems" section.

**Comment by alexis-via** [comment]
@clonedagain I took into account your remarks

**Comment by cdubuit** [comment]
Great :+1:

**Comment by alexis-via** [comment]
All remarks of the reviewers have been taken into account :+1:

**Comment by alexis-via** [comment]
Travis error is completely unrelated (res.partner.bank)

**Pull request merged**